index.html

```diff
@@ -7,5 +7,6 @@
   </head>
   <body>
     <h1>안녕 친구들</h1>
```

Comment:
이번 수정의 범위 밖이지만, 한 줄 띄웠으면 좋겠습니다. 
```suggestion
    <h1>안녕 친구들</h1>
    
```